test/crash_course_test.exs

```diff
@@ -1,8 +1,58 @@
-defmodule CrashCourseTest do
+defmodule CrashCourseAgentTest do
   use ExUnit.Case
-  doctest CrashCourse
+  doctest CrashCourseAgent
 
-  test "greets the world" do
-    assert CrashCourse.hello() == :world
+  use ExUnit.Case
+
+  test 'launches GenServer' do
+   {status, _pid} = start_server()
+   assert status == :ok
```

Comment:
One famous French pilot _Antoine de Saint-Exupéry_ once wrote: _“You become responsible, forever, for what you have tamed.”_

Stop it, please. The server I mean :)